apps/api/src/app/layouts/usecases/get-novu-layout/get-novu-layout.usecase.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import * as fs from 'fs';
+import { GetNovuLayoutCommand } from './get-novu-layout.command';
+import { ApiException } from '../../../shared/exceptions/api.exception';
+
+@Injectable()
+export class GetNovuLayout {
+  async execute(command: GetNovuLayoutCommand): Promise<string> {
+    const template = await this.loadTemplateContent('layout.handlebars');
+    if (!template) throw new ApiException('Novu default template not found');
+
+    return template;
+  }
+
+  private async loadTemplateContent(name: string) {
+    return new Promise<string>((resolve, reject) => {
+      let path = '';
+      if (!process.env.E2E_RUNNER) {
+        path = '/src/app/layouts/usecases/get-novu-layout';
+      }
+      fs.readFile(`${__dirname}${path}/templates/${name}`, (err, content) => {
```

Comment:
We can keep the async/await patterns if we use the `readFile` method from the promises library of Node.js (available since v.10; now with a different import path).
`import { readFile } from 'node:fs/promises';`

Reply:
Good catch, didn't know that!